src/acceptance-test/java/net/consensys/orion/acceptance/dsl/OrionInstance.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.orion.acceptance.dsl;
+
+import net.consensys.cava.crypto.sodium.Box;
+import net.consensys.cava.crypto.sodium.Box.PublicKey;
+import net.consensys.cava.io.Base64;
+import net.consensys.orion.http.handler.receive.ReceiveRequest;
+import net.consensys.orion.http.server.HttpContentType;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import io.vertx.core.json.JsonObject;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+/**
+ * Responsible for providing access to a running Orion instance via its HTTP interface such that payloads can be
+ * submitted and extracted.
+ *
+ * It also encapsulates all aspects of the Orion Instance - such as its datapath and maintained keys.
+ */
+public class OrionInstance {
```

Comment:
Naming; the 'Instance' part of the name seems redundant, semantically we would be creating an instance of an OrionInstance.
...but I don't have a better name, using 'Orion' would be the same name as the launcher, `OrionNode` might be a candidate 🤷‍♂

Reply:
OrionNode it is.